File: Embassat/Base/CADRootCollectionViewCell.m
//
//  CADRootCollectionViewCell.m
//  CanBuxeres
//
//  Created by Joan Romano on 01/10/13.
//  Copyright (c) 2013 Crows And Dogs. All rights reserved.
//

#import "CADRootCollectionViewCell.h"

@implementation CADRootCollectionViewCell

#pragma mark - Overridden

- (instancetype)initWithFrame:(CGRect)frame
{
    if ([super initWithFrame:frame])
    {
        [self setupView];
    }
    
    return self;
}

- (void)awakeFromNib
{
    [super awakeFromNib];
    
    [self setupView];
}

- (void)setupView{}

@end

File: Embassat/Base/CADRootCollectionViewCell.swift
//
//  CADRootCollectionViewCell.swift
//  Embassa't
//
//  Created by Joan Romano on 25/05/15.
//  Copyright (c) 2015 Crows And Dogs. All rights reserved.
//

import UIKit

public class CADRootCollectionViewCell: UICollectionViewCell {
    
    override init(frame aRect: CGRect) {
        super.init(frame: aRect)
        
        self.setupView()
    }

    required public init(coder aDecoder: NSCoder) {
        super.init(coder: aDecoder)
        
        self.setupView()
    }
    
    public override func awakeFromNib() {
        super.awakeFromNib()
        
        self.setupView()
    }
    
    public func setupView() {}
}
